prow/plugins/label/label_test.go

```diff
@@ -553,3 +552,121 @@ func TestLabel(t *testing.T) {
 		}
 	}
 }
+
+//Make sure we are repeating sig mentions for non org members (and only non org members)
+func TestRepeat(t *testing.T) {
+
+	type testCase struct {
+		name                   string
+		body                   string
+		commenter              string
+		expectedRepeatedLabels []string
+		issueLabels            []string
+		repoLabels             []string
+	}
+	testcases := []testCase{
+		{
+			name: "Dont repeat when org member adds one sig label",
+			body: "@kubernetes/sig-node-misc",
+			expectedRepeatedLabels: []string{},
+			repoLabels:             []string{"area/infra", "priority/urgent", "sig/node"},
+			issueLabels:            []string{},
+			commenter:              orgMember,
+		},
+		{
+			name: "Repeat when non org adds one sig label",
+			body: "@kubernetes/sig-node-misc",
+			expectedRepeatedLabels: []string{"@kubernetes/sig-node-misc"},
+			repoLabels:             []string{"area/infra", "priority/urgent", "sig/node", "sig/node"},
+			issueLabels:            []string{},
+			commenter:              nonOrgMember,
+		},
+		{
+			name: "Don't repeat non existent labels",
+			body: "@kubernetes/sig-node-misc @kubernetes/sig-api-machinery-bugs",
+			expectedRepeatedLabels: []string{},
+			repoLabels:             []string{},
+			issueLabels:            []string{},
+			commenter:              nonOrgMember,
+		},
+		{
+			name: "Dont repeat multiple if org member",
+			body: "@kubernetes/sig-node-misc @kubernetes/sig-api-machinery-bugs",
+			expectedRepeatedLabels: []string{},
+			repoLabels:             []string{"sig/node", "sig/api-machinery"},
+			issueLabels:            []string{},
+			commenter:              orgMember,
+		},
+		{
+			name: "Repeat multiple valid labels from non org member",
+			body: "@kubernetes/sig-node-misc @kubernetes/sig-api-machinery-bugs",
+			expectedRepeatedLabels: []string{"@kubernetes/sig-node-misc", "@kubernetes/sig-api-machinery-bugs"},
+			repoLabels:             []string{"sig/node", "sig/api-machinery"},
+			issueLabels:            []string{},
+			commenter:              nonOrgMember,
+		},
+		{
+			name: "Repeat multiple valid labels from non org member",
```

Comment:
Perhaps a better name for the test case is `Repeat multiple valid labels with a line break from non org member`.